.github/workflows/publish.yml

```diff
@@ -2,21 +2,21 @@ name: 'Publish'
 
 on:
   push:
-    branches:
-      - main
+    tags:
+    - '*'
 
 jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 10
-      - run: npm install
-      - run: npm run dev:start
-      - run: npm test
-      - uses: JS-DevTools/npm-publish@v1
-        with:
-          token: ${{ secrets.NPM_TOKEN }}
-          package: ./dist/package.json
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 10
```

Comment:
can we upgrade to v14? (the current lts)

Reply:
I was going to do a separate PR for that. the package.json lists it as node >= 13. So I was going to update it in all places at once.